samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -14,15 +14,6 @@
 
 #include "gflags/gflags.h"
 
-// gflags supports uint32 starting from v2.2 only
```

Comment:
What if user used system gflags which doesn't support uint32...

Reply:
That's why I use uint64